taxi/forms.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from .models import Driver, Car
+
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+
+
+class DriverCreationForm(UserCreationForm, forms.ModelForm):

```

Comment:
The `DriverCreationForm` should inherit only from `UserCreationForm`. Inheriting from `forms.ModelForm` is redundant because `UserCreationForm` already inherits from it.